weaver/core/drivers/fabric-driver/server/sample-satp.ts

```diff
@@ -215,102 +212,62 @@ async function assignAssetHelper(
 
     // TODO: remove the hardcoded values 
     let assignAssetRequest2 = {};
-    assignAssetRequest2['target-network'] = 'network1';
-    assignAssetRequest2['hashBase64'] = 'ivHErp1x4bJDKuRo6L5bApO/DdoyD/dG0mAZrzLZEIs=';
+    assignAssetRequest2['target-network'] = 'network2';
     assignAssetRequest2['timeout-duration'] = parseInt('3600');
     assignAssetRequest2['locker'] = 'admin';
     assignAssetRequest2['recipient'] = 'bob';
-    assignAssetRequest2['fungible'] = false;
-    assignAssetRequest2['contract-id'] = 'abc01';
-    assignAssetRequest2['hash_fn'] = '';
-    assignAssetRequest2['secret'] = 'secrettext';
-    assignAssetRequest2['param'] = 'bond01:a0demo';
+    // HACK below; reading from temp file created by the relay SATP client; ideally, we shoudld get this info from the function parameter
+    const asset_info = fs.existsSync('satp_info.txt') ? fs.readFileSync('satp_info.txt').toString() : 'bond01:a05';
+    let asset_info_parts = asset_info.split(':');
+    if (asset_info_parts.length === 1) {
+        assignAssetRequest2['param'] = asset_info_parts[0] + '_new:a05_new';
+    } else {
+        assignAssetRequest2['param'] = asset_info_parts[0] + '_new:' + asset_info_parts[1] + '_new';
+    }
     assignAssetRequest2['channel'] = 'mychannel';
     assignAssetRequest2['chaincode-id'] = 'satpsimpleasset';
 
-    const targetNetwork = assignAssetRequest2['target-network'];
-    const locker = assignAssetRequest2['locker'];
     const recipient = assignAssetRequest2['recipient'];
-    const fungible = assignAssetRequest2['fungible'];
-    const hashFn = assignAssetRequest2['hash_fn'];
-    const secret = assignAssetRequest2['secret'];
     const channel = assignAssetRequest2['channel'];
     const chaincodeId = assignAssetRequest2['chaincode-id'];
 
     let gateway: Gateway = await getNetworkGateway(networkName);
     const network: Network = await gateway.getNetwork(channel);
     const contract = network.getContract(chaincodeId);
 
-    // Hash
-    let hash: HashFunctions.Hash
-    if (hashFn === 'SHA512') {
-        hash = new HashFunctions.SHA512()
-    } else {
-        hash = new HashFunctions.SHA256()
-    }
-    hash.setPreimage(secret)
-
-    let contractId: string = null
-    if (assignAssetRequest2['contract-id']) {
-        contractId = assignAssetRequest2['contract-id']
-    }
-
     const params = assignAssetRequest2['param'].split(':')
+    const walletPath = process.env.WALLET_PATH
```

Comment:
Can you add a TODO here too, that to read recipient cert from request parameters.

Reply:
Done